cpp/src/datasets/make_regression.cu

```diff
@@ -28,9 +28,9 @@ void make_regression_helper(const cumlHandle& handle, DataT* out, DataT* values,
                             IdxT effective_rank, DataT tail_strength,
                             DataT noise, bool shuffle, uint64_t seed) {
   const auto& handle_impl = handle.getImpl();
-  cudaStream_t stream = handle_impl.getStream();
-  cublasHandle_t cublas_handle = handle_impl.getCublasHandle();
-  cusolverDnHandle_t cusolver_handle = handle_impl.getcusolverDnHandle();
+  auto stream = handle_impl.getStream();
```

Comment:
Why this change as part of this PR. IMHO it is better to keep PRs minimal. Applies to other `auto` below as well.

Reply:
ah.. this was a git revert done incorrectly. Let me fix this and remove these changes from the PR.